tools/benchmark.py

```diff
@@ -203,6 +203,27 @@ def run_http(build_dir, new_data):
     new_data["max_latency"] = {k: v["max_latency"] for k, v in stats.items()}
 
 
+def bundle_benchmark(deno_exe):
+    bundles = {
+        "file_server": "https://deno.land/std/http/file_server.ts",
+        "gist": "https://deno.land/std/examples/gist.ts",
```

Comment:
It probably makes sense to use version tagged version of URLs, otherwise benchmark will fluctuate with changes to them.